### PR TITLE
Docker development improvements and twitter cards unfurl

### PR DESCRIPTION
**Pull request by juozasg**
This merges Kingdon's fixes for the docker environment. It also introduces a default image (fluxcd.io logo) for twitter cards/unfurls.

### CODE REVIEW

**Comment by juozasg** [comment]
This is not ready to be merged yet, the logo is not well formatted I can now see using https://cards-dev.twitter.com/validator on https://deploy-preview-284--fluxcd.netlify.app/

**Comment by dholbach** [comment]
```html
<meta name=twitter:image content="https://deploy-preview-284--fluxcd.netlify.app/img/flux-icon@2x.png">
```

I wonder where this comes from ^

**Comment by stefanprodan** [comment]
This looks really bad 
Screenshot 2021-05-26 at 11 30 14

Please use the png I made for flux2 repo

**Comment by stefanprodan** [comment]
@dholbach we should set this image for twitter 
Screenshot 2021-05-26 at 11 49 54

File: layouts/partials/hooks/head-end.html [hunk 1]
```diff
@@ -6,3 +6,8 @@
 <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.css" />
 {{ end }}
 
+
+{{- template "_internal/opengraph.html" . -}}
+{{- template "_internal/google_news.html" . -}}
+{{- template "_internal/schema.html" . -}}
+{{- template "_internal/twitter_cards.html" . -}}
```
Comment (dholbach) [inline layouts/partials/hooks/head-end.html:13]: Is this required to be added? Given that it's in https://github.com/google/docsy/blob/master/layouts/partials/head.html#L19-L22 we should be good, right?

**Comment by dholbach** [comment]
Thanks for the research - looks like #286 might be minimal fix for this.

**Pull request closed without merging**